src/main/webapp/scripts/fetchData.js

```diff
@@ -2,49 +2,100 @@
         var img                     = document.getElementById('species-image');
         var pixelSlider             = document.getElementById('pixel_factor');
+        var kingdomContainer        = document.getElementById('kingdom-container');
+        var phylumContainer         = document.getElementById('phylum-container');
+        var classContainer          = document.getElementById('class-container');
+        var orderContainer          = document.getElementById('order-container');
+        var familyContainer         = document.getElementById('family-container');
+        var genusContainer          = document.getElementById('genus-container');
 
         // Update names for species
         commonNameContainer.innerText       = speciesData.commonName;
         scientificNameContainer.innerText   = speciesData.binomialName;
 
         // Map conservation status code to term and update entry
-        var statusCode  = speciesData.status;
-        statusCode      = statusCode == null ? null : statusCode.substr(0, statusCode.indexOf('['));
-        var statusMap   = {
-                            "EX" : "Extinct",
-                            "EW" : "Extinct in the Wild",
-                            "CR" : "Critically Endangered",
-                            "EN" : "Endangered",
-                            "VU" : "Vulnerable",
-                            "NT" : "Near Threatened",
-                            "LC" : "Least Concern",
-                            "DD" : "Data Deficient",
-                            "DO" : "Domesticated",
-                            "NE" : "Not Evaluated"
-                          };
-        statusContainer.innerText = statusMap[statusCode] === undefined ? "unknown" : statusCode + ": " + statusMap[statusCode];
+        var statusCode = speciesData.status;
+        statusCode = (statusCode == null) ? null : statusCode.substr(0, 2);
+        var statusMap = {
+                "EX" : "Extinct",
+                "EW" : "Extinct in the Wild",
+                "CR" : "Critically Endangered",
+                "EN" : "Endangered",
+                "VU" : "Vulnerable",
+                "NT" : "Near Threatened",
+                "LC" : "Least Concern",
+                "DD" : "Data Deficient",
+                "DO" : "Domesticated",
+                "NE" : "Not Evaluated"
+                };
+        statusContainer.innerText = (statusMap[statusCode] === undefined) ? "unknown" : statusCode + ": " + statusMap[statusCode];
 
         // Update description entry
         var notes = speciesData.wikipediaNotes;
-        descriptionContainer.innerText = notes == null ? "N/A" : notes.substr(0, notes.indexOf('['));
+        switch(notes) {
+            case null:
+                descriptionContainer.innerText = "N/A";
+                break;
+            case "":
+                descriptionContainer.innerText = "N/A";
+                break;
+            default:
+                if (notes.indexOf('[') > 0) {
```

Comment:
It was originally because wikipedia notes were formatted like "notes [citation #]" so we just wanted to get rid of the reference number. However, we actually modified it to perform that check before it even goes to datastore now so I'll delete it because it's redundant.